sdk/appconfiguration/app-configuration/package.json

```diff
@@ -53,13 +53,13 @@
     "lint": "eslint package.json api-extractor.json README.md src test --ext .ts,.javascript,.js",
     "pack": "npm pack 2>&1",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "integration-test:browser": "npm run build:test && cross-env TEST_MODE=live karma start --single-run",
-    "integration-test:node": "npm run build:test && nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace \"dist-esm/test/*.spec.js\" \"dist-esm/test/**/*.spec.js\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 180000 'dist-esm/test/**/*.spec.js'",
     "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
     "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
     "test": "npm run test:node && npm run test:browser",
-    "unit-test:browser": "",
-    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/{,!(browser)/**/}*.spec.ts\"",
+    "unit-test:browser": "echo skipped",
```

Comment:
do we have an issue to cover the browser tests?

Reply:
I found this issue for the browser tests https://github.com/Azure/azure-sdk-for-js/issues/9722

Reply:
It looks like I was mistaken then -- I assumed since we aren't recording in the browser, we weren't testing in the browser full stop (we're actually running browser tests in live mode). My bad! Can you go ahead and re-enable the browser tests, reverting my earlier suggestion?

Reply:
If you wanted to have a go at recording the browser tests (you'd need to re-enable the unit-test:browser script and go from there), that would be great too, but I don't know whether it would be straightforward. I suspect it wouldn't be too hard, but you never know.